sos/cleaner/parsers/mac_parser.py

```diff
@@ -44,7 +44,7 @@ class SoSMacParser(SoSCleanerParser):
         '534f:53'
     )
     skip_files = [
-        'sos_commands/kernel/modinfo.*'
+        'sos_commands/.*modinfo.*'
```

Comment:
Being bit paranoid, if a plugin would collect say `blah-modinfo.txt` file, we would skip cleaning it. Could be there `'sos_commands/.*/modinfo.*'` instead?

Reply:
Fair point, I'll push an update.